File: 백준/Bronze/2775. 부녀회장이 될테야/부녀회장이 될테야.swift
for _ in 0..<Int(readLine()!)! {
    let k = Int(readLine()!)!, n = Int(readLine()!)!
    var a = Array(repeating: Array(repeating: 0, count: n+1), count: k+1)
    for i in 1...n { a[0][i] = i }
    for i in 1...k {
        for j in 1...n {
            a[i][j] = a[i][j-1] + a[i-1][j]
        }
    }
    print(a[k][n])
}

File: 백준/Bronze/2775. 부녀회장이 될테야/부녀회장이 될테야.py
import sys

for _ in range(int(sys.stdin.readline())):
    k = int(sys.stdin.readline())
    n = int(sys.stdin.readline())
    apart = [[0] * (n+1) for _ in range(k+1)]
    for i in range(1, n+1):
        apart[0][i] = i
    for i in range(1, k+1):
        for j in range(1, n+1):
            apart[i][j] = apart[i][j-1] + apart[i-1][j]
    sys.stdout.write(str(apart[k][n]) + "\n")